### PR TITLE
Implementation of the PDO integration tests suite

### PR DESCRIPTION
**Pull request by gquemener**
As described in https://github.com/prolic/awesomeness/issues/4

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -15,8 +15,9 @@ services:
             - EVENTSTORE_RUN_PROJECTIONS=all
 
     postgres:
-        image: postgres:10-alpine
+        build: ./pgsql/
```
Comment (prolic) [inline docker-compose.yml]: that's not needed, you can use something like:
volumes:
- ./src/PdoEventStore/scripts/postgres:/docker-entrypoint-initdb.d

File: pgsql/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM postgres:10-alpine
```
Comment (prolic) [inline pgsql/Dockerfile]: not needed

File: pgsql/event-store-db.sql [hunk 3]
```diff
@@ -0,0 +1,56 @@
+CREATE TABLE streams (
```
Comment (prolic) [inline pgsql/event-store-db.sql]: not needed

File: phpunit.xml.dist [hunk 4]
```diff
@@ -23,10 +23,10 @@
         <env name="HTTP_USERNAME" value="admin"/>
         <env name="HTTP_PASSWORD" value="changeit"/>
         <!-- for postgres testing -->
-        <env name="PG_HOST" value="127.0.0.1"/>
+        <env name="PG_HOST" value="postgres"/>
```
Comment (prolic) [inline phpunit.xml.dist]: don't edit this here, copy to phpunit.xml instead (and put to .gitignore), change these values only for yourself.

File: phpunit.xml.dist [hunk 5]
```diff
@@ -23,10 +23,10 @@
...
         <env name="PG_PORT" value="5432"/>
-        <env name="PG_DBNAME" value="event_store_tests"/>
+        <env name="PG_DBNAME" value="event_store"/>
```
Comment (prolic) [inline phpunit.xml.dist]: I think this change here is okay, because of the default postgres script

File: phpunit.xml.dist (hunk 4 above)
Comment (prolic) [reply phpunit.xml.dist]: I will enable travis with postgres directly, no need for docker on travis

**Review by prolic — CHANGES_REQUESTED** [review]

**Review by prolic — APPROVED** [review]

**Comment by prolic** [comment]
Great @gquemener ! Thank you soooo much

**Comment by gquemener** [comment]
You're welcome, it was not finished though :thinking: :+1:

**Comment by prolic** [comment]
Ah no problem, you can push new stuff anytime.

**Pull request merged**